docs/style.md

```diff
@@ -0,0 +1,25 @@
+# Style Guide
+
+## Generic
+
+* This guide can be ignored for data files that are not to be viewed in an editor.
+* 120 character per line for all source files
+* Avoid white space errors like trailing spaces.
```

Comment:
Define all whitespace errors to avoid or point me to the section of PEP8 where this is relevant.

(If we're giving people guidelines, I think they should be specific and actionable.)

Reply:
My idea was to put it in the generic section and let PEP8 (linked below) and the Markdown guide (also linked below) handle the rest; providing a full list here may sound like it's overruling what's defined in the links below, but trailing whitespace applies to basically all languages, so i thought it might be a good example.

Reply:
This is a bit esoteric, but
* we don't want white space errors in any source file
* PEP8 only talks about python
* Markdown styleguide only talks about markdown
* Trailing whitespace is a common mistake that gets highlighted by git and github diffs

The only other common thing that comes to mind is the number of newlines at end of file.